Problem2.java

```diff
@@ -1 +1,70 @@
 
+public class priorityQueueWithLinkedList {
+
+	Node head;  
+
+	class Node {
+		int data;
+		Node next;
+		Node(int d) {
+			data = d;
+			next = null;
+		}
+	}
+	
+	public void push(int new_data) 
```

Comment:
You are not at all using priority in your code.
This wont work as priority queue.
Try again and submit PR once again.